service-storage/index.ts

```diff
@@ -0,0 +1,78 @@
+import express, { Express, Request, Response } from 'express';
+import * as fs from 'fs';
+import basicAuth from 'express-basic-auth';
+import { folderUpload, GGStorage } from './storage';
+import { getKeys, getValue, setValue } from './redis';
+import 'dotenv/config';
+
+const storageUsername = process.env.STORAGE_USERNAME || 'admin';
+const storagePassword = process.env.STORAGE_PASSWORD || 'Admin123!';
+const app: Express = express();
+const port = Number(process.env.PORT) || 5555;
+const bind = process.env.BIND || '0.0.0.0';
+
+app.use(express.json());
+app.use(
+  basicAuth({
+    users: { [storageUsername]: storagePassword },
+  })
+);
+
+app.get('/', (req: Request, res: Response) => {
+  res.send('Service Storage!');
+});
+
+app.get('/contracts/:contractAddress', async (req: Request, res: Response) => {
```

Comment:
Could we use stream in this function? it would be better memory efficient in high load service.